test/e2e/web-socket-server-url.test.js

```diff
@@ -1454,7 +1454,7 @@ describe('web socket server URL', () => {
         webSocketServer: {
           type: webSocketServer,
           options: {
-            pathname: '/custom-ws',
+            path: '/custom-ws',
```

Comment:
Yep, some of vendor modules using `path` due historical reasons instead `pathname`, maybe it will be change in future, but here we can't do something

Reply:
yep mistake here, fixed now